src/lib/y2security/selinux.rb

```diff
@@ -181,18 +182,28 @@ def mode=(id)
     # @see Yast::Bootloader#modify_kernel_params
     # @see CFA::Selinux#save
     #
-    # @return [Boolean] true if running in installation where selinux is configurable;
-    #                   false if running in installation where selinux is not configurable;
+    # @return [Boolean] true if running in installation where SELinux is configurable;
+    #                   false if running in installation where SELinux is not configurable;
     #                   the Yast::Bootloader#Write return value otherwise
     def save
       return false unless configurable?
 
+      log.info("Modifying Bootlooader kernel params using #{mode.options}")
       Yast::Bootloader.modify_kernel_params(mode.options)
+
+      log.info("Saving SELinux config file to set #{mode.id} mode")
       config_file.selinux = mode.id.to_s
       config_file.save
 
+      if Yast::Mode.installation && read_only_root_fs?
```

Comment:
what about selinux disabled. Should we still move it?

Reply:
No, I don't think so. Just I forget it. Thanks!